### PR TITLE
primeng: admin documents

### PR DESCRIPTION
**Pull request by jma**

### CODE REVIEW

File: projects/admin/src/app/record/detail-view/document-detail-view/entities-related/entities-related.component.html [hunk 1]
```diff
@@ -14,24 +14,22 @@
...
+<div class="py-2 alternate-color">
+  @for (field of entities | keyvalue; track field) {
+    <dl class="metadata">
+      <dt>{{ field.key | translate |ucfirst }}</dt>
+      <dd>
+        <ul class="list-none m-0 p-0">
           @for (entity of field.value; track entity) {
             <li>
-              <i title="{{ entity.type | translate }}" class="fa {{ entity.icon }} mr-2"></i>
+              <i title="{{ entity.type | translate }}" class="fa {{ entity.icon }}"></i>&nbsp;
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/entities-related/entities-related.component.html:25]: `[title]="entity.type | translate"`

File: projects/admin/src/app/record/detail-view/document-detail-view/files-collections/upload-files/upload-files.component.html [hunk 2]
```diff
@@ -109,39 +123,43 @@ <h5 class="mb-2" translate>Uploaded Files</h5>
...
-              aria-label="label"
-              (keyup.enter)="updateLabel(file, inputLabel.value)"
-            />
-            <div class="input-group-append">
+            <p-inputGroup>
+              <input
+                pInputText
+                #inputLabel
+                [value]="file.label"
+                (keyup)="(0)"
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/files-collections/upload-files/upload-files.component.html:133]: (0) corresponds to what ?

File: projects/admin/src/app/record/detail-view/document-detail-view/holding-organisation/holding-organisation.component.scss [hunk 3]
```diff
@@ -15,12 +15,12 @@
...
-  margin-bottom: map-get($spacers, 3);
-  padding-bottom: map-get($spacers, 1);
-  border-bottom: $border-width solid $border-color;
-}
+// .other {
+//   margin-top: map-get($spacers, 4);
+//   margin-bottom: map-get($spacers, 3);
+//   padding-bottom: map-get($spacers, 1);
+//   border-bottom: $border-width solid $border-color;
+// }
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding-organisation/holding-organisation.component.scss:26]: Delete ?

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/holding.component.ts [hunk 4]
```diff
@@ -26,9 +26,9 @@ import { forkJoin } from 'rxjs';
 import { ItemRequestComponent } from '../item-request/item-request.component';
 
 @Component({
-  selector: 'admin-document-holding',
+  selector: 'admin-document-holding, [admin-document-holding]',
   templateUrl: './holding.component.html',
-  styles: ['a.collapse-link i { min-width: 16px}']
+  styles: []
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding/holding.component.ts:31]: Remove

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/serial-holding-item/serial-holding-item.component.html [hunk 5]
```diff
@@ -30,109 +29,94 @@
...
+            <dt translate>Temporary location</dt>
+            <dd>
+              {{ item.metadata.temporary_location.pid | getRecord: 'locations' : 'field': 'name' | async }}
+            </dd>
+          </dl>
+        }
+        <!-- ITEM IN COLLECTION -->
+        @if (permissions && item.metadata.pid | itemInCollection | async; as collections) {
+          <dl class="metadata">
+            <dt>Exhibition/course</dt>
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding/serial-holding-item/serial-holding-item.component.html:110]: Missing `translate`

File: projects/admin/src/app/record/detail-view/document-detail-view/item-request/item-request.component.html [hunk 6]
```diff
@@ -14,60 +14,73 @@
...
+          showTransitionOptions="0ms"
+        />
+        <p-button
+          outlined
           [routerLink]="['/records', 'items', 'detail', recordPid]"
-          (click)="closeModal()" translate>Edit queue</button>
-        </div>
+          (click)="closeModal()" translate
+        >
+          Edit queue
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/item-request/item-request.component.html:41]: Transfer to `[label]="'Edit queue' | translate"`

File: projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.ts [hunk 7]
```diff
@@ -98,41 +77,24 @@ export class ItemDetailViewComponent implements DetailRecord, OnInit, OnDestroy
...
 
-  /** Permissions */
-  permissions: IPermissions = PERMISSIONS;
-  permissionOperator = PERMISSION_OPERATOR;
+  // /** OnInit hook */
+  // ngAfterViewInit(): void {
+  //   this.subscription.add(
+  //     this.recordService.getRecord('locations', this.record.metadata.location.pid, 1).subscribe(data => this.location = data)
+  //   );
+  // }
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.ts:91]: Remove

File: projects/admin/src/app/record/operation-logs/operation-logs.component.ts [hunk 8]
```diff
@@ -63,7 +60,7 @@ export class OperationLogsComponent implements OnInit {
     return this.recordTotals > 0 && ((this.page * this.itemsPerPage) < this.recordTotals);
   }
 
-  /** Hidden operation logs counter */
+  /** Hidden operation logs coun  ter */
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/operation-logs/operation-logs.component.ts:63]: Typo

File: projects/shared/src/lib/component/remote-search/remote-search.component.ts [hunk 9]
```diff
@@ -35,12 +35,12 @@ export class RemoteSearchComponent implements OnInit, OnDestroy {
 
   // You must use lowercase variable names for this to work in a web component.
   // Use @Input in this case, as the web component does not work with input (signal).
-  @Input() admin: boolean | string = true;
+  admin: boolean;
```
Comment (Garfield-fr) [inline projects/shared/src/lib/component/remote-search/remote-search.component.ts:38]: make this after input (above hideSearchElement)

**Review by Garfield-fr — APPROVED** [review]

File: projects/admin/src/app/record/detail-view/document-detail-view/files-collections/upload-files/upload-files.component.html (hunk 2 above)
Comment (jma) [reply projects/admin/src/app/record/detail-view/document-detail-view/files-collections/upload-files/upload-files.component.html]: Removed

**Pull request merged**